src/subarraynew.jl

```diff
@@ -0,0 +1,36 @@
+typealias NullableSubArray{T,N,P<:NullableArray,IV,LD} SubArray{T,N,P,IV,LD}
+
+@inline function Base.isnull(V::NullableSubArray, I::Real...)
+    @boundscheck checkbounds(V, I...)
+    @inbounds return V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
```

Comment:
Perfect

On Monday, April 4, 2016, Scott P. Jones <notifications@github.com> wrote:

> In src/subarraynew.jl
> <https://github.com/JuliaStats/NullableArrays.jl/pull/101#discussion_r58378574>
> :
>
> > @@ -0,0 +1,36 @@
> > +typealias NullableSubArray{T,N,P<:NullableArray,IV,LD} SubArray{T,N,P,IV,LD}
> > +
> > +@inline function Base.isnull(V::NullableSubArray, I::Real...)
> > +    @boundscheck checkbounds(V, I...)
> > +    @inbounds return V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
>
> @kmsquire <https://github.com/kmsquire> I was asked to keep the fixing of
> the indentation as a separate commit, and I think the fixes related to
> adding Compat should probably be in a separate commit from the fixes
> related to JuliaLang/julia#15071
> <https://github.com/JuliaLang/julia/pull/15071>, but I'll squash out that
> commit. Does that sound reasonable?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaStats/NullableArrays.jl/pull/101/files/a2a4740c12a904f5bd11672d1190ef0eca59b02f..6a849974f7af5b75df58727dbe9f22dc5b978960#r58378574>
>